src/App/MappedName.h

```diff
@@ -474,6 +474,28 @@ class AppExport MappedName
         return appendToBuffer(res, startPosition, len);
     }
 
+    const char * toString(std::string &s, int from=0, int len=-1) const
```

Comment:
Needs a better name, and a clang-tidy pass.

Reply:
What if we call it appendToBuffer and then note that the method right below it already exists?  Ooops.